File: advancedscreenswitchersample/src/main/java/me/aartikov/advancedscreenswitchersample/screens/InnerScreen.java
package me.aartikov.advancedscreenswitchersample.screens;

import java.io.Serializable;

import me.aartikov.alligator.Screen;

/**
 * Date: 22.01.2016
 * Time: 0:38
 *
 * @author Artur Artikov
 */
public class InnerScreen implements Screen, Serializable {
	private int mCounter;

	public InnerScreen(int counter) {
		mCounter = counter;
	}

	public int getCounter() {
		return mCounter;
	}
}

File: flowscreenswitchersample/src/main/java/me/aartikov/flowscreenswitchersample/screens/InnerScreen.kt
package me.aartikov.flowscreenswitchersample.screens;

import java.io.Serializable;

import me.aartikov.alligator.Screen;

public class InnerScreen implements Screen, Serializable {
    private int mCounter;

    public InnerScreen(int counter) {
        mCounter = counter;
    }

    public int getCounter() {
        return mCounter;
    }
}
